package.json

```diff
@@ -64,21 +67,19 @@
     "eslint-plugin-react": "^7.23.2",
     "full-icu": "^1.3.4",
     "gh-pages": "^3.1.0",
-    "html-webpack-plugin": "^5.3.1",
     "husky": "^6.0.0",
     "jest": "^26.6.3",
     "lint-staged": "^11.0.0",
+    "parcel": "^2.0.0-beta.2",
     "prettier": "^2.3.0",
     "react": "^17.0.2",
     "react-dom": "17.0.2",
-    "react-hot-loader": "^4.13.0",
+    "rollup": "^2.52.7",
```

Comment:
Is rollup required for parcel? Or is this leftover?

Reply:
So the way I've done it is rollup will be used to build the package, and parcel will be used to start/build the demo app.

It doesn't necessarily need to be these tools, it's just that they provide the simplest approach.

Reply:
Right, just wanted to make sure it was required. Thanks